docs/rst/spelling_wordlist.txt

```diff
@@ -32,6 +32,7 @@ Redistributable
 Requiredness
 runtime
 scalable
+validator
```

Comment:
I don't see this word used in the documentation. Removing it the docs compile.

Reply:
Weird, it fails for me when not in the list.